financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -147,6 +172,14 @@ async function getUniswapPairDetails(web3, syntheticTokenAddress, collateralCurr
   return {};
 }
 
+async function createBalancerPriceFeedForEmp(logger, web3, networker, getTime, empAddress, config = {}) {
+  assert(empAddress, "createUniswapPriceFeedForEmp: Must pass in an `empAddress`");
+  const emp = getEmpAtAddress(web3, empAddress);
+  const balancerTokenOut = await emp.methods.collateralCurrency().call();
```

Comment:
I dont think this works. Take for example yUSD. For this contract the synthetic is `yUSD-SEP20` and the collateral currency is `WETH`. However, the pool has `yUSD-SEP20` and `USDC` in it, not WETH. You cant always assume that the pool will have the same currency in it as the collateral currency.

Clearly, this is fine if the user has provided their own config as they'll be able to specify the `in` and out `tokens` but I'm not sure if this logic in here makes sense as it's not super trivial to infer this.

Reply:
this is really serious thanks for catching that

Reply:
So would the `balancerAddress` that the user passes in reference the yUSD-USDC pool?

Reply:
I think im going to force the user to specify the `tokenOut` and infer `tokenIn` from the EMP.  Basically if using balancer, throw an error if the balancerTokenOut is not specified, which in the yusd case would be the usdc address.

Reply:
So the EMP's synth token will always be `tokenIn`? Just want to double check that this assumption is possible and that we won't need to pass in an `inverted` boolean

Reply:
I dont know if that is the correct design, but yes, synth will be tokenIn. I feel like i need a full integration test to check that is right. I am also assuming we can invert by flipping tokenin/tokenout in the balancer call.